sql/dql/basic-selects/select-cols.md

```diff
@@ -28,19 +28,40 @@ aspects:
 ---
 ## Content
 
+Although returning all data from a table is commonly used when inspecting new databases, it is advised against using the `SELECT *` syntax for *real production data* (which usually contains a large amount of information) due to the following reasons:
```

Comment:
kind of verbose introduction.
can start with something like this:
> You can control what fields are returned in the resulting set when using the `SELECT` statement...

Reply:
moved paragraph to the end of the insight